tests/test_exiftool_direct.py

```
@@ -0,0 +1,79 @@
+    # Créer une image de test
+    temp_dir = Path(tempfile.mkdtemp())
+    media_path = temp_dir / "test.jpg"
+    img = Image.new('RGB', (100, 100), color='blue')
+    img.save(media_path)
+    
+    try:
+        print("=== TEST EXIFTOOL DIRECT ===\n")
+        
+        # 1. Ajouter des keywords initiaux
+        print("1. Ajout keywords initiaux...")
+        subprocess.run([
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)